contracts/most/UtmostInterface.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @title An interface for Utmost contract.
+ */
+interface UtmostInterface {
+
+    /* External Functions */
+
+    /**
+     * @notice Mints the given amount of token to beneficiary.
+     *
+     * @param _beneficiary Address of beneficiary where tokens are minted.
+     * @param _amount Amount in wei.
+     *
+     * @return bool `true` if success else `false`.
+     */
+    function mint(
+        address payable _beneficiary,
+        uint256 _amount
+    )
+        external
+        returns(bool);
```

Comment:
is `mint` and `burn` not part of the Utility token interface?

Reply:
so this should either be just `UtilityTokenI.sol` or build on that

Reply:
Renamed to `UtilityTokenInterface` with `mint` and `burn`. `BurnFrom` will be added with `CoConsensus::withdraw` Pr.

Reply:
> is mint and burn not part of the Utility token interface?
> so this should either be just UtilityTokenI.sol or build on that

Here I am guilty for the naming: `UtmostInterface.sol` or `UtmostI.sol`

The idea behind the name`UtmostInterface` used here was that,
we realized that the `UtilityToken` was redundant for `Utmost` because the functions of the `UtilityToken` will never be called, so the dependencies of the `UtilityToken` will be removed from the `Utmost`. `Utmost` can be considered as a special version of `UtilityToken`. The `UtilityToken` could be removed from the code and can be reintroduced later when we actually need it. So the interface was named `UtmostInterface`.

But I am fine with `UtilityTokenI.sol`